scripts/multicluster/setup.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# This script sets up a demo multicluster environment for OSM development.
+# It deploys the sample bookstore application on cluster c1 and c2. C2 exports its
+# bookstore service via ServiceExport and c1 imports it via ServiceImport. A new EndpointSlice
+# resource on c1 is created to store accessible endpoint from c2.
+
+# shellcheck disable=SC2128
+# shellcheck disable=SC2086
+cd "$(dirname ${BASH_SOURCE})"
+set -e
+
+c1=${CLUSTER1:-c1}
+c2=${CLUSTER2:-c2}
+k1="kubectl --context ${c1}"
+k2="kubectl --context ${c2}"
+app_port=14001
+
+# ==== 1. Setup CRDs
+
+${k1} apply -f ./crd -f ./rbac
+${k2} apply -f ./crd -f ./rbac
+
+# ==== 2. Deploy applications
+for ctx in ${c1} ${c2}; do
+    kubectl config use-context "${ctx}"
+
+    echo "[${ctx}] Installing osm"
+    osm install
+
+    echo "[${ctx}] Deploying demo application"
+    kubectl create namespace bookstore
+    kubectl create namespace bookbuyer
+    kubectl create namespace bookthief
+    kubectl create namespace bookwarehouse
+    osm namespace add bookstore bookbuyer bookthief bookwarehouse
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/release-v1.2/manifests/apps/bookbuyer.yaml
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/release-v1.2/manifests/apps/bookstore.yaml
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/release-v1.2/manifests/apps/bookwarehouse.yaml
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/release-v1.2/manifests/apps/mysql.yaml
+done
+
+# ==== Deploy a curl client for testing
+${k1} apply -f - <<EOF
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: curl
+  namespace: bookbuyer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: curl
+  template:
+    metadata:
+      labels:
+        app: curl
+    spec:
+      containers:
+      - name: curl
+        image: curlimages/curl
+        command: ["sleep", "1000000"]
+EOF
+
+# ==== 3. Create ServiceExport
+${k2} apply -f - <<EOF
+apiVersion: multicluster.x-k8s.io/v1alpha1
+kind: ServiceExport
+metadata:
+  name: bookstore
+  namespace: bookstore
+EOF
+
+# ==== 4. Create ServiceImport
+
+# create a meta-service to get a valid local service IP
+${k1} apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: mcs-bookstore
+  namespace: bookstore
```

Comment:
let's create this in a non-meshed namespace so it doesn't conflict with anything else in OSM with permissive mode